Libraries/Image/Image.ios.js

```diff
@@ -105,6 +134,24 @@ var Image = React.createClass({
     validAttributes: ReactIOSViewAttributes.UIView
   },
 
+  onLoadingStart: function(event: Event) {
+    if (this.props.onLoadingStart) {
+      this.props.onLoadingStart(merge(event.nativeEvent, { target: this }));
```

Comment:
can you use the non standard 

```javascript
this.props.onLoadingStart({...event.nativeEvent, target: this});
```

we use this instead of merge in fb codebase, it is much easier to write and read :)